src/status_im2/contexts/quo_preview/list_items/preview_lists.cljs

```diff
@@ -104,10 +104,9 @@
 
 (defn view
   []
-  (let [state (reagent/atom {:type               :accounts
-                             :size               :size/s-32
-                             :number             4
-                             :more-than-99-label "99+"})
```

Comment:
is there more uses of this in the codebase? `:more-than-99-label` or perhaps it's for another component?

Reply:
There was only one usage in `switcher-card`. It's removed 👍 .

The `more-than-99-label` key is a custom label for the overflow item. But, it's not needed as the number tag component has replaced it (and as per the design system).